src/ews_xml.erl

```diff
@@ -86,6 +116,31 @@ to_string(Name) when is_atom(Name) -> atom_to_list(Name);
 to_string(Name) when is_binary(Name) -> binary_to_list(Name);
 to_string(Name) when is_list(Name) -> Name.
 
+%% Escape special characters `\r` `<' and `&', flattening the text.
+%% Also escapes `>', just for symmetry.
+
+export_text(T) ->
+    export_text(T, []).
+
+export_text([$\r | T], Cont) ->
+    "&#13;" ++ export_text(T, Cont);
+export_text([$< | T], Cont) ->
+    "&lt;" ++ export_text(T, Cont);
+export_text([$> | T], Cont) ->
+    "&gt;" ++ export_text(T, Cont);
+export_text([$& | T], Cont) ->
+    "&amp;" ++ export_text(T, Cont);
+export_text([C | T], Cont) when is_integer(C) ->
+    [C | export_text(T, Cont)];
+export_text([T | T1], Cont) ->
+    export_text(T, [T1 | Cont]);
```

Comment:
What is this case for? `binary_to_list` returns `[byte()]` so the case above should always hit before this one, no?

Reply:
The code is from OTP actually. I think this is for io_lists, and it puts things in a buffer and does the buffer at a later stage with the clause below.